src/Asset.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+/*
+ * This file is part of the limit0/assets package.
+ *
+ * (c) Limit Zero, LLC <contact@limit0.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Limit0\Assets;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
```

Comment:
It's up to you... would be nice to not need the dep on Symfony. How hard would it be to ditch?